frontend/benefit/applicant/src/components/applications/forms/application/step5/useApplicationFormStep5.ts

```diff
@@ -96,7 +97,12 @@ const useApplicationFormStep5 = (
 
   const handleSubmit = (): void => {
     const submitFields = isSubmit
-      ? { status: APPLICATION_STATUSES.RECEIVED }
```

Comment:
It's getting a bit complicated with inner resolving logic, I perhaps would call `resolveStatus` or some other method here, or use `React.useMemo` to resolve the status value, outside the handleSubmit method.

Reply:
its not reusable, only local logic, so separate method is a bit strange at least